slack-server-integration-common/src/main/java/com/atlassian/plugins/slack/api/webhooks/AppHomeOpenedSlackEvent.java

```diff
@@ -9,17 +9,32 @@
 /*
 https://api.slack.com/events/app_home_opened
 {
-  "token": "XXYYZZ",
-  "team_id": "TXXXXXXXX",
-  "api_app_id": "AXXXXXXXXX",
-  "event": {
     "type": "app_home_opened",
-    "user": "UFXXXXXXX",
-    "channel": "DLX1Z611B"
-  },
-  "type": "event_callback",
-  "event_id": "EvLX1P57NC",
-  "event_time": 1564526855
+    "user": "U061F7AUR",
```

Comment:
Note we had actual IDs redacted before. They are not really a secret like a password but giving away internal IDs may facilitate an attacker that's trying to gather information about our internal systems.
I also realize not all IDs were redacted so... 🤷‍♂️